lib/emque/consuming/adapters/rabbit_mq/worker.rb

```diff
@@ -73,11 +73,33 @@ def process_message(delivery_info, metadata, payload)
               )
               ::Emque::Consuming::Consumer.new.consume(:process, message)
               channel.ack(delivery_info.delivery_tag)
-            rescue StandardError
-              channel.nack(delivery_info.delivery_tag)
+            rescue StandardError => ex
+              if retryable_errors.any? { |error| ex.class.to_s =~ /#{error}/ }
```

Comment:
Can this section of code in the rescue block be extracted to its own method/class to simplify a bit?